mapeditor/resourceExtractor/ResourceConverter.cpp

```diff
@@ -0,0 +1,91 @@
+#include "StdInc.h"
+
+#include "ResourceConverter.h"
+
+#include "../lib/JsonNode.h"
+#include "../lib/VCMIDirs.h"
+#include "../lib/filesystem/Filesystem.h"
+
+#include "BitmapHandler.h"
+#include "Animation.h"
+
+#include "boost/filesystem/path.hpp"
+#include "boost/locale.hpp"
+
+namespace bfs = boost::filesystem;
+
+// converts all pcx files from /Images into PNG
+void ConvertPcxToPng(bool deleteOriginals)
+{
+	bfs::path imagesPath = VCMIDirs::get().userCachePath() / "extracted" / "Images";
```

Comment:
pathToQString(VCMIDirs::get().userCachePath()) - and then you can work with QString/QFile/QFilename/QDir

Reply:
I am not that familiar with QT. Since we also use boost can I leave it like it is now?

Reply:
IMO, editor shouldn't have dependency on boost